recipes/qt/all/conanfile.py

```diff
@@ -0,0 +1,673 @@
+
+        if self.options.qtmultimedia:
+            args.append("--alsa=" + ("yes" if self.options.with_libalsa else "no"))
+
+        for opt, conf_arg in [
+                              ("with_doubleconversion", "doubleconversion"),
+                              ("with_freetype", "freetype"),
+                              ("with_harfbuzz", "harfbuzz"),
+                              ("with_libjpeg", "libjpeg"),
+                              ("with_libpng", "libpng"),
+                              ("with_sqlite3", "sqlite")]:
+            if getattr(self.options, opt):
+                if self.options.multiconfiguration:
+                    args += ["-qt-" + conf_arg]
+                else:
+                    args += ["-system-" + conf_arg]
+            else:
+                args += ["-no-" + conf_arg]
+
+        libmap = [("zlib", "ZLIB"),
+                  ("openssl", "OPENSSL"),
+                  ("pcre2", "PCRE2"),
+                  ("glib", "GLIB"),
+                  # ("libiconv", "ICONV"),# QTBUG-84708
+                  ("double-conversion", "DOUBLECONVERSION"),
+                  ("freetype", "FREETYPE"),
+                  ("fontconfig", "FONTCONFIG"),
+                  ("icu", "ICU"),
+                  ("harfbuzz", "HARFBUZZ"),
+                  ("libjpeg", "LIBJPEG"),
+                  ("libpng", "LIBPNG"),
+                  ("sqlite3", "SQLITE"),
+                  ("libmysqlclient", "MYSQL"),
+                  ("libpq", "PSQL"),
+                  ("odbc", "ODBC"),
+                  ("sdl2", "SDL2"),
+                  ("openal", "OPENAL"),
+                  ("zstd", "ZSTD"),
+                  ("libalsa", "ALSA"),
+                  ("xkbcommon", "XKBCOMMON")]
+        for package, var in libmap:
+            if package in self.deps_cpp_info.deps:
+                if package == 'freetype':
+                    args.append("\"%s_INCDIR=%s\"" % (var, self.deps_cpp_info[package].include_paths[-1]))
+
+                args.append("\"%s_LIBS=%s\"" % (var, " ".join(self._gather_libs(package))))
+
+        for package in self.deps_cpp_info.deps:
+            args += ["-I " + s for s in self.deps_cpp_info[package].include_paths]
+            args += ["-D " + s for s in self.deps_cpp_info[package].defines]
+            args += ["-L " + s for s in self.deps_cpp_info[package].lib_paths]
+
+        if 'libmysqlclient' in self.deps_cpp_info.deps:
+            args.append("-mysql_config " + os.path.join(self.deps_cpp_info['libmysqlclient'].rootpath, "bin", "mysql_config"))
+        if 'libpq' in self.deps_cpp_info.deps:
+            args.append("-psql_config " + os.path.join(self.deps_cpp_info['libpq'].rootpath, "bin", "pg_config"))
+        if self.settings.os == "Macos":
+            args += ["-no-framework"]
+        elif self.settings.os == "Android":
+            args += ["-android-ndk-platform android-%s" % self.settings.os.api_level]
+            args += ["-android-abis %s" % {"armv7": "armeabi-v7a",
```

Comment:
Qt supports multi abis since 5.14, sadly without https://github.com/conan-io/conan/issues/6384 implemented that feature can't be used.